edk2toolext/capsule/signtool_signer.py

```diff
@@ -20,6 +20,10 @@
 from edk2toollib.utility_functions import RunCmd
 
 GLOBAL_SIGNTOOL_PATH = None
+SUPPORTED_SIGNTOOL_FORMAT_OPTIONS = {
```

Comment:
I should probably rename SIGNTOOL to SIGNATURE... as I think this was supposed to be a signature_options['format_options'] ...

Reply:
Addressed in commit 2